src/main/java/model/Lotto.java

```diff
@@ -0,0 +1,54 @@
+package model;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lotto {
+
+    // 로또 번호 관련 상수 선언
+    public static final int LOTTO_MIN_NUMBER = 1;
+    public static final int LOTTO_MAX_NUMBER = 45;
+    public static final int LOTTO_CREATE_SIZE = 6;
+    public static final int LOTTO_PRICE = 1000;
+    public static final List<Integer> LOTTO_NUMBER_POOL =
+            IntStream
+            .rangeClosed(LOTTO_MIN_NUMBER,LOTTO_MAX_NUMBER)
+            .boxed()
+            .collect(Collectors.toList());
```

Comment:
```suggestion
    public static final int LOTTO_MIN_NUMBER = 1;
    public static final int LOTTO_MAX_NUMBER = 45;
    public static final int LOTTO_CREATE_SIZE = 6;
    public static final int LOTTO_PRICE = 1_000;
    public static final List<Integer> LOTTO_NUMBER_POOL =
            IntStream
            .rangeClosed(LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER)
            .boxed()
            .collect(Collectors.toList());
```

- `_`를 사용하여 가독성을 높일 수 있어요!
- 공백도 컨벤션입니다! 커밋 전 코드 정렬을 습관화한다면 놓치지 않을 수 있을 것 같아요 😄

Reply:
리뷰 감사합니다 !
> * `_`를 사용하여 가독성을 높일 수 있어요!
-> int타입을 상수화할 때, `_` 문자를 사용할 수 있는지 몰랐는데 알려주셔서 감사합니다. 

> * 공백도 컨벤션입니다! 커밋 전 코드 정렬을 습관화한다면 놓치지 않을 수 있을 것 같아요 😄
-> 네 ! 코드 정렬 단축키 'cmd + opt + l' 새겨놓고 커밋하겠습니다 :D